### DISCUSSION TITLE
小爱音箱app打开不了了

### DISCUSSION THREAD

**Discussion by rgh27** [kkkgo/PaoPaoDNS#224] [问答和帮助]
之前是可以的，最近发现xiaomusic登陆不了了，然后发现小爱音箱app也进不去了，用手机流量就可以进了，这是什么原因呢？

**Comment by kkkgo** [comment]
你应该尝试切换DNS而不是切换网络才能说明问题。

**Comment by rgh27** [comment]
好吧，是网络的问题，重启下主路由就好了
